src/WKWebKit/Defs.cs

```diff
@@ -101,7 +106,7 @@ public enum WKDataDetectorTypes : nuint {
 #endif
 	}
 
-	[iOS (10,0)][Mac (10,12, only64: true)]
+	[iOS (10,0)][Mac (10,12, onlyOn64: true)]
```

Comment:
how did that compile ? :|

Reply:
PMCS "fixed" it before it hit a compiler.

Reply:
:lol: